packages/vscode-extension/src/common/Project.ts

```diff
@@ -28,7 +28,7 @@ export type ProjectState = {
     | "bootError"
     | "runtimeError"
     | "bundleError"
-    | "incrementalBundleError"
+    | "bundlingError"
```

Comment:
we can't have both `bundleError` and `bundlingError`, how do you tell which one means what?